apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -342,15 +346,40 @@ export class CaseService {
           address: this.config.email.replyToEmail,
         },
         to,
-        subject: this.formatMessage(m.signedRuling.subject, {
-          courtCaseNumber,
-        }),
+        subject,
         text: stripHtmlTags(body),
         html: body,
         attachments,
       })
     } catch (error) {
       this.logger.error('Failed to send email', { error })
+
+      this.eventService.postErrorEvent(
+        'Failed to send email',
+        {
+          courtCaseNumber,
+          subject,
+          to: Array.isArray(to)
+            ? to
+                .reduce(
```

Comment:
I was a bit confused about what this code did until I wrote it in a terminal and saw what it did with my own eyes. I was mainly confused about the `.slice(2)` part. I think it would be clearer to include a index parameter to the reduce function and use that to do what slice does in this version, i.e.

`[].reduce((acc, recipient, index) => `${index > 0 ? `${acc}, ` : ''}${recipient.name} (${recipient.address})`, '')`

What do you think?